### DISCUSSION TITLE
Searching a symbol in all dependencies

### DISCUSSION THREAD

**Discussion by lispercat** [nvim-java/nvim-java#166] [General]
What would be a command similar to IntelliJ when I search all my packages for some symbol using Ctrl-Alt-Shift N?

**Comment by neandrake** [comment]
@lispercat are you referring to searching types including everything imported through dependencies?

**Comment by s1n7ax** [comment]
@lispercat Try `Telescope lsp_workspace_symbols query=Something`

**Comment by s1n7ax** [comment]
![image](https://github.com/nvim-java/nvim-java/assets/18459807/ecd4f50d-692e-42c6-be0e-d9948e3c0a30)

**Comment by s1n7ax** [comment]
https://github.com/nvim-java/nvim-java/wiki/Tips-&-Tricks#workspace-symbols-search
